bench/chromium/corpus_mem.py

```
@@ -0,0 +1,1489 @@
+            raise errors[0]
+
+
+def spawn_in_cgroup(cg_path, argv, log_path, env=None):
+    """Launch argv with the launching shell moved into cg_path first.
+
+    cgroup membership is inherited across fork and survives the user-namespace
+    and uid transitions fcvm makes, so every descendant lands in the same leaf.
+    """
+    script = f'sudo -n sh -c "echo $BASHPID > {cg_path}/cgroup.procs" || exit 90; exec "$0" "$@"'
+    handle = open(log_path, "wb")
+    return subprocess.Popen(["bash", "-c", script] + list(argv),
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)